.ci/integrationTestSelector.groovy

```diff
@@ -155,19 +207,18 @@ pipeline {
   }
 }
 
-def wrappingup(){
+def wrappingup(Map params = [:]){
+  def isJunit = params.containsKey('isJunit') ? params.get('isJunit') : true
```

Comment:
There are not tests for the opbeans yet, so this is the way we can reuse the same method without the junit step.